creusot-contracts/src/std/vec.rs

```diff
@@ -74,15 +74,15 @@ extern_spec! {
                 fn pop(&mut self) -> Option<T>;
 
                 #[requires(@ix < (@self).len())]
-                #[ensures(result == (@self)[@ix])]
+                #[ensures(result == self[@ix])]
```

Comment:
@jhjourdan my objective is fixing issues like this, and here we can also remove the `@` in front of the `ix`.